README.md

```diff
@@ -150,6 +150,18 @@ Access tokens may be provided with one of the following:
 - Query string parameter `token`.
 
 
+#### Identity / claims
+
+A valid token is exposed to a controller as `current_token` and identity information
+can be extracted by providing a claim name through hash-syntax:
```

Comment:
Bit unsure whether to call this hash-syntax, index/subscript operator or something else entirely.